src/br/com/zup/nossacasacodigo/author/Author.java

```diff
@@ -0,0 +1,54 @@
+package br.com.zup.nossacasacodigo.author;
+
+public class Author {
+	// nome, email e uma descri��o
```

Comment:
_Deixamos pistas que facilitem o uso do código onde não conseguimos resolver com compilação._

Achei bem legal que você comenta o código!

Em geral, comentários podem ser utilizados para dar mais semântica, explicar o porquê de um trecho, ou serem utilizados como forma de documentação. [Aqui tem um texto bacana, mas em inglês, sobre comentários](https://refactoring.guru/smells/comments). 

O Alberto tem um vídeo legal sobre o assunto também: https://youtu.be/7g4aeMKwGcs

O que acha de revisitar os comentários do seu código e ver lugares assim que, talvez, possam ficar sem comentários?